manageiq-providers-hawkular.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,lib}/**/*"]
 
-  s.add_runtime_dependency "hawkular-client", "~> 3.0.2"
+  s.add_runtime_dependency "hawkular-client", "~> 4.0.0"
```

Comment:
Created this PR to also change version number on manageiq-providers-kubernetes: https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/56